packages/fela-dom/src/dom/rehydrate.js

```diff
@@ -18,7 +18,7 @@ export default function rehydrate(renderer: DOMRenderer): void {
   render(renderer)
 
   const idQuery =
-    renderer.rendererId.length > 0
+    renderer.rendererId && renderer.rendererId.length > 0
       ? `[data-fela-id="${renderer.rendererId}"]`
       : ''
```

Comment:
See #681, we should always add `data-fela-id`

Reply:
We still need it for the edge case where ppl use different versions. Shouldn't happen but might be^^